types/store/shapediverStorePlatformSavedStates.ts

```diff
@@ -91,4 +91,27 @@ export type IShapeDiverStorePlatformSavedStateExtended =
 		TSavedStateQueryItem,
 		TSavedStateQueryPropsExt,
 		SavedStateCacheKeyEnum
-	>;
+	> & {
+		/** The initial saved state to be applied on load, indexed by session ID */
+		initialSavedState: Record<string, TSavedStateData | undefined>;
+		/** Set the initial saved state for a specific session */
+		setInitialSavedState: (
+			sessionId: string,
+			savedState: TSavedStateData | undefined,
+		) => void;
+		/** Handle initial saved state from query parameter */
+		handleInitialSavedState: (sessionId: string) => Promise<void>;
```

Comment:
@LemToUp this can be removed, it won't be needed once the implementation is done as described here: https://github.com/shapediver/AppBuilderShared/pull/243/commits/8fdfc82507213b2ae1e9ea682643cb34bbdac6ab

Reply:
Relevant for the: https://github.com/shapediver/AppBuilderShared/pull/243#discussion_r2541127523

Reply:
Done.